src/encord_active/lib/project/project.py

```diff
@@ -317,6 +362,19 @@ def split_lr_video(label_row: LabelRow, project_file_structure: ProjectFileStruc
         du = label_row.data_units[data_hash]
         suffix = f".{du['data_type'].split('/')[1]}"
         video_path = (lr_structure.images_dir / du["data_hash"]).with_suffix(suffix)
-        slice_video_into_frames(video_path)
+        sliced_frames, _ = slice_video_into_frames(video_path)
+
+        # 'create_many' is not available for SQLite in prisma
```

Comment:
[nitpick] 💅

We could probably use `batch` create instead 🙂

Reply:
fixed, we would want to change the way we have structured the download process as it make more sense to batch in groups of label row + its data units, so we ensure the integrity of the label rows contents in the db.